src/kem/kyber/ind_cpa.rs

```diff
@@ -309,8 +311,11 @@ pub(crate) fn decrypt<
         .chunks_exact((COEFFICIENTS_IN_RING_ELEMENT * VECTOR_U_COMPRESSION_FACTOR) / 8)
         .enumerate()
     {
-        let u = deserialize_little_endian::<VECTOR_U_COMPRESSION_FACTOR>(u_bytes);
-        u_as_ntt[i] = ntt_representation(decompress(u, 10));
+        let u = decompress(
```

Comment:
Lets be consistent and make `bits_per_compressed_coefficient` a const generic on this function. I forgot that and only did it in compress. But we should handle the two the same.